element.io/nightly/config.json

```diff
@@ -49,5 +49,6 @@
         "feature_spotlight": true,
         "feature_video_rooms": true
     },
-    "map_style_url": "https://api.maptiler.com/maps/streets/style.json?key=fU3vlMsMn4Jb6dnEIFsx"
+    "map_style_url": "https://api.maptiler.com/maps/streets/style.json?key=fU3vlMsMn4Jb6dnEIFsx",
+    "tray_icons": {}
```

Comment:
we shouldn't be supplying default values in the config. This is because the code needs to handle a case where the value isn't supplied, and we pick sensible defaults for everything. Whenever we can, we force ourselves to test this by not specifying them in the production configs.

Reply:
Am I reading this right if with this change, you now consider this resolved?

Reply:
well, it's resolved when the config.json files for nightly and release aren't modified :p